scripts/exregional_run_analysis.sh

```diff
@@ -365,42 +365,62 @@ sed -i "s/hh/${HH}/"     coupler.res
 # copy observation files to working directory 
 #
 #-----------------------------------------------------------------------
-obs_source=rap
-if [[ ${HH} -eq '00' || ${HH} -eq '12' ]]; then
-  obs_source=rap_e
-fi
+if [[ "${NET}" = "RTMA"* ]]; then
+  SUBH=$(date +%M -d "${START_DATE}")
+  obs_source="rtma_ru"
+  case $MACHINE in
+  "WCOSS_C" | "WCOSS" | "WCOSS_DELL_P3")
```

Comment:
should not have two piece of case $MACHINE

Reply:
I will remove the machine dependency in the RTMA if block